tests/sandbox/create_message_batches/test_field_validation.py

```diff
@@ -338,7 +410,19 @@ def test_invalid_dob(nhsd_apim_proxy_url, dob, correlation_id):
 @pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
 def test_invalid_routing_plan(nhsd_apim_proxy_url, correlation_id):
     """
-    .. py:function:: Test invalid routing plans
+    .. py:function:: Scenario: A user submitting a request with an invalid \
+       routing plan receives a 400 'Invalid Value' response
+
```

Comment:
Explain it needs to be a UUID - link to wikipedia page on UUID like in our spec.